.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,33 @@
+name: publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: main
```

Comment:
Should / can we remove these two lines? I guess we would like to run the tests on the branch dictated by the release? I'm guessing this forces the tests to run on main branch.

Reply:
Hmn, I think it would be clearer if we removed them.

I can't recall why they have been setup in the first place, but we'll add them back if something breaks.